src/ILCompiler/ILCompiler.sln

```diff
@@ -58,14 +58,14 @@ Global
 		{DD5B6BAA-D41A-4A6E-9E7D-83060F394B10}.Debug|x86.Build.0 = Debug|x86
 		{DD5B6BAA-D41A-4A6E-9E7D-83060F394B10}.Release|Any CPU.ActiveCfg = Release|Any CPU
 		{DD5B6BAA-D41A-4A6E-9E7D-83060F394B10}.Release|Any CPU.Build.0 = Release|Any CPU
-		{DD5B6BAA-D41A-4A6E-9E7D-83060F394B10}.Release|arm.ActiveCfg = Debug|arm
```

Comment:
This came with https://github.com/jkotas/corert/commit/df3351af7151cc356697c460ed9a019cf9982125#diff-03b5feef38e0ee0285d44462b4f3c8f9. It does not look intentional.